frontend/src/common/utils/captureSentryError.ts

```diff
@@ -0,0 +1,23 @@
+import * as Sentry from '@sentry/react';
+
+interface CaptureSentryErrorParams {
+  error: unknown;
+  level: 'warning' | 'error';
+  feature: string;
+  step: string;
+}
+
+export const captureSentryError = ({
+  error,
+  level,
+  feature,
+  step,
+}: CaptureSentryErrorParams) => {
+  Sentry.captureException(error, {
+    level: level,
+    tags: {
+      feature: feature,
+      step: step,
+    },
```

Comment:
p5: 객체 생성할 때 동일한 key에 값이 들어가야 한다면 속성 축약 표현으로 해도 될 거 같아! 👍
```suggestion
    level,
    tags: {
      feature,
      step,
    },
```

Reply:
[refactor: 동일한 key,value 속성 축약 표현 적용](https://github.com/woowacourse-teams/2025-Fit-toring/pull/519/commits/d3b5166ea874cb89b8e579361105680aa41d6667) 👍👍👍